server/src/main/java/org/moparforia/server/net/packethandlers/golf/ChatHandler.java

```diff
@@ -56,7 +56,8 @@ public boolean handle(Server server, Packet packet, Matcher message) {
                 }
             }
         } else if (message.group(2).equals("command")) {
-            //todo
+            System.out.println("Unhandled lobby command: " + message.group());
+            // TODO
```

Comment:
I would prefer to have a `// TODO: handle lobby commands` or similar here instead of a simple `// TODO`

Reply:
Fixed